app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,32 @@
+<nav class="bg-slate-400 fixed z-20 w-full">
+    <ul class="m-2 flex gap-5 items-center">
+        <li>
+            <%= link_to "Public recipes", "/public_recipes", class: 'px-4 py-2 border-black cursor:pointer'  %>
+        </li>
+        <%if user_signed_in?%>
+            <li>
+                <%= link_to "My recipes", recipes_path%>
+                <%# My recipes %>
+            </li>
+            <li>
+                <%= link_to "My Foods", foods_path%>
+                <%# My Foods %>
+            </li>
+            <li>
+                <%= link_to "General shopping list", "/general_shopping_list"%>
+                <%# My Foods %>
+            </li>
+            <li>
+                <%# <%=link_to 'Sign in', "users/sign_out"%>
```

Comment:
- [ ]  Please kindly delete the commented lines of code to keep your code more clean 🙃